src/features/simulation/components/form/CreateMovementSimulationStrategyForm.tsx

```diff
@@ -12,85 +12,141 @@ import {
   FormBox, FormInpiutField, ElementBox, FormSelect,
 } from './common';
 
-export const movementSimulationStrategyInitialValues = {
+type InitialValues = {
+  type: string,
+  slowDownProbability: string,
+  maxVelocity: string,
+  threshold?: string,
+  accelerationDelayProbability?: string,
+  breakLightReactionProbability?: string
+};
+
+export const movementSimulationStrategyInitialValues: InitialValues = {
   type: 'NAGEL_SCHRECKENBERG',
   slowDownProbability: '',
   maxVelocity: '',
 };
 
-
 interface Props {
   compatibleStrategies: MovementSimulationStrategyType[],
+  values: InitialValues;
 }
 
 const labels: Map<MovementSimulationStrategyType, string> = new Map([
   ['NAGEL_SCHRECKENBERG', 'Nagel Schreckenberg'],
   ['MULTI_LANE_NAGEL_SCHRECKENBERG', 'Multi Lane Nagel'],
+  ['BRAKE_LIGHT', 'Brake Light'],
 ]);
 
-function CreateMovementSimulationStrategyForm({ compatibleStrategies }: Props): JSX.Element {
+function CreateMovementSimulationStrategyForm({ compatibleStrategies, values }: Props): JSX.Element {
   return (
         <div>
             <h1>Movement Simulation Strategy</h1>
```

Comment:
wraspnij w memo bo bedzie lagowac (patrz inne formy) + moze wydzielic render formow brake lighta do zmienneuj? bo sie cos duze to drzewo robi

Reply:
ale to chyba jest w memo tak jak inne

Reply:
true